Sources/NIOTransportServices/NIOTSEventLoop.swift

```diff
@@ -150,8 +174,89 @@ internal class NIOTSEventLoop: QoSEventLoop {
     }
 
     func preconditionInEventLoop(file: StaticString, line: UInt) {
+        // Do not forward to the other overload.
+        dispatchPrecondition(condition: .onQueue(self.loop))
+    }
+
+    func preconditionInEventLoop(_ message: @autoclosure() -> String, file: StaticString, line: UInt) {
```

Comment:
Hmmmmm. I don't know that it's possible for us to meet the criteria for safely implementing this method correctly: specifically, we need to use only public API. I'm open to evolving it, but for now I think the safest thing to do is to simply discard the message. I know it's not good, but it's the only choice we have, and the absence of a debugging message, while highly annoying, is probably acceptable if the crash report is intact.